### DISCUSSION TITLE
Can playlist state be exposed?

### DISCUSSION THREAD

**Discussion by dcastillo3** [the-maazu/react-material-music-player#31] [Ideas]
I would like to support CTA based on player state.

Example:
PlayerInterface.getTrack([]) // returns playing track data

**Comment by the-maazu** [comment]
Yes this is doable. I plan on exposing the full state in beta channel v1.2. Will work on this soon.

**Reply by ary1111** [reply]
@the-maazu would this include getting the current/remaining track time?

**Reply by the-maazu** [reply]
getState() and subscribe() added to interface in beta channel.
